contracts/JBProjects.sol

```diff
@@ -38,27 +37,21 @@ contract JBProjects is ERC721Votes, Ownable, IJBProjects, JBOperatable {
   */
   uint256 public override count = 0;
 
-  /**
-    @notice
-    The contract resolving each project id to its ERC721 URI
-    
-    @dev
-    This is optional for each project
-  */
-  IJBTokenUriResolver public tokenUriResolver;
-
   /** 
     @notice 
-    The IPFS CID for each project, which can be used to reference the project's metadata.
-
-    @dev
-    This is optional for each project.
+    The metadata for each project, which can be used across several domains.
 
-    _projectId The ID of the project to which the URI belongs.
+    _projectId The ID of the project to which the metadata belongs.
     _domain The domain within which the metadata applies.
   */
   mapping(uint256 => mapping(uint256 => string)) public override metadataContentOf;
 
+  /**
+    @notice
+    The contract resolving each project ID to its ERC721 URI.
+  */
+  IJBTokenUriResolver public override tokenUriResolver;
```

Comment:
For consistency should we s/Uri/URI everywhere (contract name, local variable, etc.)?